src/pages/[platform]/start/getting-started/data-model/index.mdx

```diff
@@ -440,7 +440,7 @@ import {
   ReactiveFormsModule,
   Validators
 } from '@angular/forms';
-import { generateClient } from 'aws-amplify/api';
+import { generateClient, type Client } from 'aws-amplify/api';
```

Comment:
Is `type` necessary here? TS should know this is a type, and this reads weird.

```suggestion
import { generateClient, Client } from 'aws-amplify/api';
```

Reply:
I initially had it without but @renebrandel made this suggestion. I agree with it as it makes it clear to the reader it's a type being imported and not the client.

Reply:
Just realized my feedback is the opposite of @renebrandel's. 🤣 I've usually only seen this like:
```
import type { Client } from 'aws-amplify/api';
```
Is it valid to put it inside the curly braces like this?

Reply:
Yeah it was introduced in typescript@4.5 looks like

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-5.html#type-modifiers-on-import-names